tiny_dnn/util/parameter_init.h

```diff
@@ -53,10 +53,10 @@ class constant : public scalable {
   constant() : scalable(float_t{0}) {}
   explicit constant(float_t value) : scalable(value) {}
 
-  void fill(parameter *p, size_t fan_in, size_t fan_out) override {
+  void fill(Parameter *parameter, size_t fan_in, size_t fan_out) override {
```

Comment:
Besides, `TitleCase` for class names and `snake_case` for method names can still be doable. `TitleCase` gives me ability to name `Parameter parameter` while `parameter` class name won't...